virttest/utils_test/__init__.py

```diff
@@ -421,6 +421,19 @@ def get_memory_info(lvms):
     return meminfo
 
 
+def find_python(session, try_binaries=['python3', 'python']):
```

Comment:
I'd suggest adding `python2` as well (or instead of python), because when one won't install python3 but installs python2 on newer distros, python is not created while python2 is.

Reply:
so let me re-phrase it, would you please change `python` to `python2`? It should exist on older as well as newer distros, while python might not.

Reply:
Another issue, please use tuple instead of mutable list as the list might get modified....

Reply:
@ldoktor yes, thanks for the suggestions. I will implement both.